src/API.jl

```diff
@@ -49,22 +64,31 @@ const ATTRIBUTES_MAPPINGS = Dict{String,String}(
   "indeterminateicon" => "indeterminate-icon",
   "indeterminatevalue" => "indeterminate-value",
   "inputclass" => "input-class",
+  "inputdebounce" => "input-debounce",
   "inputstyle" => "input-style",
   "insetlevel" => "inset-level",
   "itemaligned" => "item-aligned",
   "keepcolor" => "keep-color",
   "labelalways" => "label-always",
   "labelcolor" => "label-color",
+  "labellines" => "label-lines",
   "labelslot" => "label-slot",
   "labelvalue" => "label-value",
-  "labelvalueleft" => "left-label-value",
```

Comment:
These are breaking changes - please revert. And please keep the logical hierarchy `label > property > attribute`